README.md

```diff
@@ -2,36 +2,35 @@
 <img src="http://cdn2.eegeo.com/wp-content/uploads/2017/04/WRLD_Blue.png"  align="right" height="80px" />
 </a>
 
-# WRLD Unity AR Samples
+# WRLD Unity XR Samples
 
 ![WRLD](http://cdn2.eegeo.com/wp-content/uploads/2017/04/screenselection01.png)
 
-* [Support](#support)
-* [Getting Started](#getting-started)
-* [WRLD Unity SDK Documentation](#wrld-sdk-documentation)
-* [License](#license)
+- [WRLD Unity XR Samples](#wrld-unity-xr-samples)
+  - [Support](#support)
+  - [Getting Started](#getting-started)
+  - [WRLD Unity SDK Documentation](#wrld-unity-sdk-documentation)
+  - [License](#license)
 
-These sample apps showcase [WRLD Unity SDK's](https://wrld3d.com/unity/latest/docs/api/) integration with ARCore and ARKit platforms to build basic AR experiences for supported Android and iOS devices.
+This sample app showcase [WRLD Unity SDK's](https://wrld3d.com/unity/latest/docs/api/) integration with Unity's AR Foundation and XR Management plugins to build basic XR experiences for supported devices.
 
 ## Support
 
-If you have any questions, bug reports, or feature requests, feel free to submit to the [issue tracker](https://github.com/wrld3d/wrld-unity-ar-samples/issues) for this repository.
+If you have any questions, bug reports, or feature requests, feel free to submit to the [issue tracker](https://github.com/wrld3d/wrld-unity-xr-samples/issues) for this repository.
 
 ## Getting Started
 
-These sample apps demonstrate the use of the WRLD's Unity SDK that allows developers to create AR experiences for supported Android and iOS devices.
+This sample app demonstrate the use of the WRLD's Unity SDK that allows developers to create XR experiences for supported devices.
 
-Choose a platform from the table below to see detailed instructions for building each app.
+Choose a platform below to see detailed instructions for building each app.
```

Comment:
Replace with something like: "See our [documentation pages](combined doc page URL) for detailed instructions for building the apps"